code/modules/clothing/gloves/tacklers.dm

```diff
@@ -67,6 +67,20 @@
 	resistance_flags = NONE
 	strip_mod = 1.2 // because apparently black gloves had this
 
+/obj/item/clothing/gloves/tackler/combat/goliath
+	name = "goliath gloves"
+	desc = "Rudimentary combat gloves. To be fair, Goliaths are really good at grappling... Also, they're fireproof."
```

Comment:
These gloves will probably become really usefull for miners unless you make them unable to use modern guns, which would make sense that they couldnt. Because big fingers, just make it add that trait when worn
```suggestion
	desc = "Rudimentary tackling gloves. The goliath hide makes it great for grappling with targets, while also being fireproof."
```